src/clarity-angular/layout/vertical-nav/vertical-nav-icon.directive.spec.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) 2016-2017 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+
+import {Component} from "@angular/core";
+import {VerticalNavService} from "./providers/vertical-nav.service";
+import {ComponentFixture, TestBed} from "@angular/core/testing";
+import {ClrIconModule} from "../../icon/icon.module";
+import {ClrVerticalNavModule} from "./vertical-nav.module";
+export default function (): void {
+    describe("Vertical Nav Icons", () => {
+        let fixture: ComponentFixture<any>;
+
+        beforeEach(() => {
+            TestBed.configureTestingModule({
+                imports: [
+                    ClrVerticalNavModule,
```

Comment:
great advice. I think I am even going to get rid of `clr-icon` and not import `ClrIconModule` because the only thing I am testing here is the `clrVerticalNavIcon` directive which isn't dependent on `clr-icon`